iothub/service/src/Common/Exceptions/ServerBusyException.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.Azure.Devices.Common.Exceptions
     /// The exception that is thrown when the IoT Hub is busy.
     /// </summary>
     /// <remarks>
-    /// This exception typically means the service is unavilable due to high load or an unexpected error and is usually transient. The best course of action is to retry your operation after some time.
+    /// This exception typically means the service is unavailable due to high load or an unexpected error and is usually transient. The best course of action is to retry your operation after some time. By default, the SDK will utilize the <see cref="ExponentialBackoff"/> retry strategy.
```

Comment:
The service doesn't use a retry strategy. This type is in the device client, and my local build fails due to this reference.

Reply:
Oh this was the resolution of a rebase. I think someone added it previously and it got to my branch.